mm2src/mm2_rpc/src/data/legacy/utility.rs

```diff
@@ -1,14 +1,30 @@
-use derive_more::Display;
+use rpc::v1::types::H256 as H256Json;
 use serde::{Deserialize, Serialize};
 
-#[derive(Serialize, Deserialize, Display)]
-#[serde(rename_all = "lowercase")]
-pub enum Status {
-    Success,
-}
-
 #[derive(Serialize, Deserialize)]
 pub struct MmVersionResponse {
     pub result: String,
     pub datetime: String,
 }
+
+#[derive(Default, Serialize)]
+#[serde(tag = "method", rename = "stop")]
+pub struct StopRequest {}
+
+#[derive(Default, Serialize)]
+#[serde(tag = "method", rename = "version")]
+pub struct VersionRequest {}
+
+#[derive(Serialize, Deserialize)]
+#[serde(tag = "method", rename = "ban_pubkey")]
+pub struct BanPubkeysRequest {
+    pub pubkey: H256Json,
+    pub reason: String,
+}
+
+#[derive(Serialize, Deserialize)]
+#[serde(tag = "type", content = "data")]
+pub enum UnbanPubkeysReq {
+    All,
+    Few(Vec<H256Json>),
```

Comment:
`Some(Vec<H256Json>)`

Reply:
I'm sorry, `Some` could be interpreted wrong by Rust developers, may be that a reason why it was named like that originally